_rules/focusable-no-keyboard-trap-80af7b.md

```diff
@@ -68,45 +68,45 @@ _There are no major accessibility support issues known for this rule._
 
 #### Passed Example 1
 
-No trap for keyboard navigation.
+These [focusable][] elements do not create a trap for keyboard navigation.
 
 ```html
 <a href="#">Link 1</a> <button>Button1</button>
 ```
 
 #### Passed Example 2
 
-Using `tabindex="1"`.
+This [focusable][] element does not create a trap for keyboard navigation.
```

Comment:
I don't think example descriptions should repeat. That suggests to me they're not specific enough.

Reply:
Revised the descriptions